examples/custom-pipeline/README.md

```diff
@@ -22,25 +22,42 @@ This is an example of how to use the gllm-plugin library to build a simple pipel
      > For Windows, please make sure to check the `Add python.exe to PATH` option during the installation process.
    </details>
 
-2. **Access to GDP-ADMIN/gen-ai-internal repository**
+2. **Access to GDP-ADMIN/gen-ai-internal and GDP-ADMIN/gen-ai-external repositories**
 
-   You can try to access the [GDP-ADMIN/gen-ai-internal](https://github.com/GDP-ADMIN/gen-ai-internal) repository in your browser. If you don’t have access, please make a request to ticket(at)gdplabs.id.
+   You can try to access the [GDP-ADMIN/gen-ai-internal](https://github.com/GDP-ADMIN/gen-ai-internal)  and [GDP-ADMIN/gen-ai-external](https://github.com/GDP-ADMIN/gen-ai-external) repositories in your browser. If you don’t have access, please make a request to ticket(at)gdplabs.id.
 
 3. **SSH Key in your GitHub Account**
 
    You must add your SSH key to your GitHub account. Please follow this instruction by GitHub: [Adding a new SSH key to your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account). This is required as this sample has dependency to a private GitHub repository.
 
-4. **VSCode IDE**
 
-   - Go to [VSCode](https://code.visualstudio.com/download) to download VSCode IDE.
+## Running the Code
 
+1. Configure environment variables: copy `.env.example` to `.env` and set up your `OPENAI_API_KEY` and `LANGUAGE_MODEL`.
 
-## Running the Code
+   - For Linux, macOS, or Windows WSL:
+
+     ```bash
+     cp .env.example .env
+     ```
 
-1. Execute the script:
+2. Execute the script:
 
    - For Linux, macOS, or Windows WSL:
      ```bash
      ./local-start.sh
```

Comment:
As we will ask PM to try this, we need to add `local-start.bat` as well. Unless we don't plan to ask PM to try this example(?)